samples/helm-deploy/templates/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: carbon-aware-sdk
+  namespace: {{ .Values.namespace }}
```

Comment:
We should use `.Release.namespace` rather than the value from values.yaml. https://helm.sh/docs/chart_template_guide/builtin_objects/